code-buddy.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Student Companion App</title>
+  </head>
+  <body>
+    <header class="headersec">
+      <img
+        class="headersec__background"
+        src="/icons/background_shape_1.svg"
+        alt=""
+      />
+      <h1 class="headersec__title">Code Buddys</h1>
+      <h3 class="headersec__subtitle">on Monday &#8212; 18.07.2020</h3>
```

Comment:
Hierarchically this `h3`-element should be a `h2`. If this is too big, then you should change the styling via CSS. 
```suggestion
      <h2 class="headersec__subtitle">on Monday &#8212; 18.07.2020</h2>
```